dev-env/lambda_functions/pdf_text_extract/app.py

```
@@ -0,0 +1,73 @@
+
+import json
+import boto3
+import io
+from pypdf import PdfReader as Re
+from common.ingest import ingest_extracted_text
+
+
+
+def extract_text(file_stream):
+    """Extract text from a PDF file stream and return as a string."""
+    try:
+        reader = Re(file_stream)
+    except Exception as e:
+        print(f"Error reading PDF file: {e}")
+        return None
```

This should probably return an error code instead of None

---

I'll make sure to update that tomorrow morning.